src/ajax/load.js

```diff
@@ -64,7 +66,7 @@ jQuery.fn.load = function( url, params, callback ) {
 				// Otherwise use the full result
 				responseText );
 
-		}).complete( callback && function( jqXHR, status ) {
+		}).always( callback && function( jqXHR, status ) {
```

Comment:
We could really use some comments here, because the code takes advantage of some tricky behavior. If the request succeeds, this function gets `( data, status, jqXHR )`, but they are ignored because `response` was set above. If it fails, this function gets `( jqXHR, status, error )` (the first two of which coincide with `complete`) and are used in the body. It's probably also worth documenting that the `self.each` takes advantage of `.each`'s undocumented second parameter.